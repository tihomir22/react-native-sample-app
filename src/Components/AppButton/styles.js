import {StyleSheet, Platform} from 'react-native';
import {Metrics, Adjust, Fonts, Colors} from '../../utils';
export default StyleSheet.create({
  buttonMainContainer: {
    width: '100%',
    justifyContent: 'center',
    alignItems: 'center',
    display: 'flex',
  },
  buttonContainer: {
    width: 236,
    height: 52,
    justifyContent: 'center',
    alignItems: 'center',
    display: 'flex',
    borderRadius: 50,
    backgroundColor: '#ffa185',
  },
  buttonLabel: {
    fontSize: 26,
    color: 'white',
    fontFamily: 'BryantPro-Bold',
  },
});
