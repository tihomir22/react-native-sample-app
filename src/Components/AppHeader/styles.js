import {StyleSheet, Platform} from 'react-native';
import {Metrics, Adjust, Fonts, Colors} from '../../utils';
export default StyleSheet.create({
  headerContainer: {
    width: '100%',
    ...Platform.select({
      ios: {
        paddingTop:
          Metrics.screenHeight === 812 ||
          Metrics.screenWidth === 812 ||
          Metrics.screenHeight === 896 ||
          Metrics.screenWidth === 896
            ? 45
            : 25,
        height:
          Metrics.screenHeight === 812 ||
          Metrics.screenWidth === 812 ||
          Metrics.screenHeight === 896 ||
          Metrics.screenWidth === 896
            ? Metrics.tabBarHieghtIphoneX
            : Metrics.tabBarHeight,
      },
      android: {
        height: Metrics.tabBarHeightAndroid,
        paddingTop: 15,
      },
    }),
    backgroundColor: '#025ef8',
    justifyContent: 'space-between',
    alignItems: 'center',
    flexDirection: 'row',
    display: 'flex',
    paddingHorizontal: 17,
    elevation: 2,
    zIndex: 101,
  },
  leftIcon: {
    width: 20,
    height: 20,
    marginRight: 10,
  },
  rightIcon: {
    width: 54,
    height: 54,
  },
  rightIconContainer: {
    width: 54,
    height: 54,
    justifyContent: 'center',
    alignItems: 'flex-start',
  },
  leftIconContainer: {
    width: 54,
    height: 54,
    justifyContent: 'center',
    alignItems: 'flex-end',
  },

  headerTitle: {
    fontSize: 24,
    fontFamily: 'Roboto-Medium',
    color: 'white',
    fontWeight: '600',
  },
  labelStylesDash: {
    fontSize: 25,
    fontFamily: 'Roboto-Medium',
    color: 'white',
    marginLeft: 35,
  },
});
