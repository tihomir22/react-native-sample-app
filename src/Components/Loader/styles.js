import {StyleSheet, Platform} from 'react-native';

export default StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: 'rgba(0,0,0,0.5)',
  },
  loaderImage: {
    width: 80,
    height: 80,
    resizeMode:'contain'
  },
});
