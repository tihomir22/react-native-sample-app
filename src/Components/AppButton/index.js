import React from 'react';
import {View, Text, TouchableOpacity} from 'react-native';
import styles from './styles';
export default class AppButton extends React.Component {
  render() {
    return (
      <View
      allowFontScaling={false}
        style={[
          styles.buttonMainContainer,
          this.props.buttonMainContainerStyles,
        ]}>
        <TouchableOpacity
          onPress={this.props.onPress}
          style={styles.buttonContainer}>
          <Text style={styles.buttonLabel}>{this.props.label}</Text>
        </TouchableOpacity>
      </View>
    );
  }
}
