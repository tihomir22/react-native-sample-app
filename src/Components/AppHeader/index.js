import React from 'react';
import {View, Text, Image, TouchableOpacity} from 'react-native';
import styles from './styles';
import {Images} from '../../utils';
const AppHeader = ({
  showLeftIcon = true,
  title,
  rightIcon,
  rightIconStyle,
  onPressLeft,
  onPressRight,
}) => {
  return (
    <View style={styles.headerContainer}>
      <View style={styles.leftIconContainer}>
        {showLeftIcon ? (
          <TouchableOpacity
            onPress={onPressLeft}
            style={styles.leftIconContainer}>
            <Image style={styles.leftIcon} source={Images.left} />
          </TouchableOpacity>
        ) : null}
      </View>
      <Text
        allowFontScaling={false}
        numberOfLines={1}
        style={styles.headerTitle}>
        {title}
      </Text>
      <View style={styles.rightIconContainer}>
        {rightIcon ? (
          <TouchableOpacity
            style={styles.rightIconContainer}
            onPress={onPressRight}>
            <Image style={rightIconStyle} source={rightIcon} />
          </TouchableOpacity>
        ) : null}
      </View>
    </View>
  );
};
export default AppHeader;
