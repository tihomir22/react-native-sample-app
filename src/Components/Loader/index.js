import React from 'react';
import {View, ActivityIndicator, Modal, Image} from 'react-native';
import styles from './styles';
import {Images, Colors} from '../../utils';
export default Loader = props => {
  return (
    <Modal visible={props.isVisible} transparent>
      <View style={[styles.container, props.style]}>
        {props.hide ? (
          <View />
        ) : (
          <Image style={styles.loaderImage} source={Images.loader} />
        )}
      </View>
    </Modal>
  );
};