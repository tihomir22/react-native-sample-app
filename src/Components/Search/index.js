import React from 'react';
import { View, TextInput, Image } from 'react-native';
import styles from './styles';
import { Images } from '../../utils';

export default (Search = (props) => {
	return (
		<View style={styles.container}>
			<View style={styles.SectionStyle}>
				<Image source={Images.search} style={styles.icon} />
				<TextInput
					ref={props.ref}
					allowFontScaling={false}
					onSubmitEditing={props.onSubmitEditing}
					editable={true}
					selectTextOnFocus={true}
					onChangeText={props.onChangeText}
					placeholderTextColor={'#808080'}
					value={props.value}
					style={styles.input}
					placeholder="Search Property"
				/>
			</View>
		</View>
	);
});
