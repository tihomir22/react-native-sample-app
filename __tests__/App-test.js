import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
// import App from '../App';
import Search from '../src/Components/Search/index';
import AppButton from '../src/Components/AppButton/index';
import AppHeader from '../src/Components/AppHeader/index';
// import Contacts from '../src/Components/Contacts/index';
// import Documents from '../src/Components/Documents/index';
// import DocumentView from '../src/Components/DocumentView/index';
// import ImportantDateItem from '../src/Components/ImportantDateItem/index';
// import ImportantDates from '../src/Components/ImportantDates/index';
import Loader from '../src/Components/Loader/index';
import ShareUpdate from '../src/Containers/ShareUpdate';

// import SliderItem from '../src/Components/SliderItem/index';
// import Status from '../src/Components/Status/index';

global.fetch = jest.fn(() => new Promise(resolve => resolve()));
jest.mock('react-native-gesture-handler', () => {});
jest.mock('@react-navigation/stack', () => {});
jest.mock('@react-native-community/async-storage');

it('SearchBar Renders Correctly', () => {
  const tree = renderer.create(<Search />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('AppButton Renders Correctly', () => {
  const tree = renderer.create(<AppButton />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('AppHeader Renders Correctly', () => {
  const tree = renderer.create(<AppHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});

const shareUpdateParam = {
  params: {
    documents: [],
    data: {id: '', address: ''},
    token: '',
  },
};
it('ShareUpdate Renders Correctly', () => {
  const tree = renderer
    .create(<ShareUpdate route={shareUpdateParam} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

// it('Contacts Renders Correctly', () => {
//   const tree = renderer.create(<Contacts />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

// it('Documents Renders Correctly', () => {
//   const tree = renderer.create(<Documents />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

// it('DocumentView Renders Correctly', () => {
//   const tree = renderer.create(<DocumentView />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

// it('ImportantDateItem Renders Correctly', () => {
//   const tree = renderer.create(<ImportantDateItem />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

// it('ImportantDates Renders Correctly', () => {
//   const tree = renderer.create(<ImportantDates />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

it('Loader Renders Correctly', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();

  // });it('SliderItem Renders Correctly', () => {
  //   const tree = renderer.create(<SliderItem />).toJSON();
  //   expect(tree).toMatchSnapshot();

  // });it('Status Renders Correctly', () => {
  //   const tree = renderer.create(<Status />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});
