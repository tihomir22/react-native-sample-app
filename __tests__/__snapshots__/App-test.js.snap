// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppButton Renders Correctly 1`] = `
<View
  allowFontScaling={false}
  style={
    Array [
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffa185",
        "borderRadius": 50,
        "display": "flex",
        "height": 52,
        "justifyContent": "center",
        "opacity": 1,
        "width": 236,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "BryantPro-Bold",
          "fontSize": 26,
        }
      }
    />
  </View>
</View>
`;

exports[`AppHeader Renders Correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#025ef8",
      "display": "flex",
      "elevation": 2,
      "flexDirection": "row",
      "height": 80,
      "justifyContent": "space-between",
      "paddingHorizontal": 17,
      "paddingTop": 25,
      "width": "100%",
      "zIndex": 101,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "height": 54,
        "justifyContent": "center",
        "width": 54,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "height": 54,
          "justifyContent": "center",
          "opacity": 1,
          "width": 54,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/left.png",
          }
        }
        style={
          Object {
            "height": 20,
            "marginRight": 10,
            "width": 20,
          }
        }
      />
    </View>
  </View>
  <Text
    allowFontScaling={false}
    numberOfLines={1}
    style={
      Object {
        "color": "white",
        "fontFamily": "Roboto-Medium",
        "fontSize": 24,
        "fontWeight": "600",
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "height": 54,
        "justifyContent": "center",
        "width": 54,
      }
    }
  />
</View>
`;

exports[`Loader Renders Correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0,0,0,0.5)",
          "flex": 1,
          "justifyContent": "center",
        },
        undefined,
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/images/loader1.gif",
        }
      }
      style={
        Object {
          "height": 80,
          "resizeMode": "contain",
          "width": 80,
        }
      }
    />
  </View>
</Modal>
`;

exports[`SearchBar Renders Correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f6f8fd",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFF",
        "borderColor": "#dfdfdf",
        "borderRadius": 50,
        "borderWidth": 2,
        "flexDirection": "row",
        "height": 45,
        "margin": 10,
        "width": "85%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/images/search.png",
        }
      }
      style={
        Object {
          "alignItems": "center",
          "height": 25,
          "margin": 8,
          "padding": 14,
          "resizeMode": "stretch",
          "width": 25,
        }
      }
    />
    <TextInput
      allowFontScaling={false}
      editable={true}
      placeholder="Search Property"
      placeholderTextColor="#808080"
      rejectResponderTermination={true}
      selectTextOnFocus={true}
      style={
        Object {
          "flex": 1,
          "fontFamily": "Avenir",
          "fontSize": 17,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;

exports[`ShareUpdate Renders Correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#f5f8fd",
      "display": "flex",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#025ef8",
        "display": "flex",
        "elevation": 2,
        "flexDirection": "row",
        "height": 80,
        "justifyContent": "space-between",
        "paddingHorizontal": 17,
        "paddingTop": 25,
        "width": "100%",
        "zIndex": 101,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "height": 54,
          "justifyContent": "center",
          "width": 54,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "flex-end",
            "height": 54,
            "justifyContent": "center",
            "opacity": 1,
            "width": 54,
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../src/assets/images/left.png",
            }
          }
          style={
            Object {
              "height": 20,
              "marginRight": 10,
              "width": 20,
            }
          }
        />
      </View>
    </View>
    <Text
      allowFontScaling={false}
      numberOfLines={1}
      style={
        Object {
          "color": "white",
          "fontFamily": "Roboto-Medium",
          "fontSize": 24,
          "fontWeight": "600",
        }
      }
    >
      Share Update
    </Text>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "height": 54,
          "justifyContent": "center",
          "width": 54,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#83b1ff",
        "justifyContent": "center",
        "paddingVertical": 15,
        "width": "100%",
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Object {
          "alignSelf": "center",
          "color": "#FFF",
          "fontFamily": "Roboto-Medium",
          "fontSize": 19,
          "textAlign": "center",
          "width": "70%",
        }
      }
    >
      
    </Text>
    <Text
      allowFontScaling={false}
      style={
        Object {
          "alignSelf": "center",
          "color": "#FFF",
          "fontFamily": "Roboto-Medium",
          "fontSize": 19,
          "textAlign": "center",
          "width": "70%",
        }
      }
    >
      undefined, undefined, undefined
    </Text>
  </View>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "paddingVertical": 20,
      }
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        style={
          Object {
            "display": "flex",
            "flexDirection": "row",
            "flexWrap": "wrap",
            "width": 690,
          }
        }
      />
      <View
        style={
          Array [
            Object {
              "height": 90,
              "width": 690,
            },
            Object {
              "zIndex": 5000,
            },
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onLayout={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomLeftRadius": 5,
              "borderBottomRightRadius": 5,
              "borderColor": "#dfdfdf",
              "borderTopLeftRadius": 5,
              "borderTopRightRadius": 5,
              "borderWidth": 1,
              "flex": 1,
              "flexDirection": "row",
              "opacity": 1,
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#868686",
                  "fontSize": 20,
                },
                Object {
                  "color": "#CECECE",
                  "fontSize": 20,
                },
                Object {
                  "flex": 1,
                  "opacity": 1,
                },
                Object {
                  "marginLeft": 5,
                  "marginRight": 0,
                },
                false,
                Object {
                  "marginLeft": 5,
                },
              ]
            }
          >
            
          </Text>
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "borderBottomLeftRadius": 0,
                  "borderTopLeftRadius": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "paddingVertical": 8,
                },
              ]
            }
          >
            <View
              style={
                Array [
                  Object {},
                  Object {
                    "opacity": 1,
                  },
                ]
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#CECECE",
                      "fontSize": 15,
                    },
                    undefined,
                    Object {
                      "fontFamily": "Feather",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomLeftRadius": 5,
                "borderBottomRightRadius": 5,
                "borderColor": "#dfdfdf",
                "borderTopLeftRadius": 5,
                "borderTopRightRadius": 5,
                "borderWidth": 1,
                "paddingHorizontal": 10,
                "paddingVertical": 5,
              },
              Object {
                "alignItems": "center",
                "borderTopLeftRadius": 0,
                "borderTopRightRadius": 0,
                "justifyContent": "center",
                "position": "absolute",
                "width": "100%",
              },
              Object {
                "backgroundColor": "#fafafa",
              },
              Object {
                "borderWidth": 0,
                "display": "none",
                "position": "relative",
              },
              Object {
                "maxHeight": 500,
                "top": undefined,
                "zIndex": 5000,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "width": "100%",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              defaultValue={null}
              onChangeText={[Function]}
              placeholder="Search for an item"
              placeholderTextColor="gray"
              rejectResponderTermination={true}
              style={
                Array [
                  Object {
                    "borderBottomWidth": 1,
                    "borderColor": "#dfdfdf",
                    "flex": 1,
                    "marginBottom": 10,
                    "paddingHorizontal": 0,
                    "paddingVertical": 8,
                  },
                  Object {
                    "color": "#868686",
                    "fontSize": 20,
                  },
                ]
              }
              underlineColorAndroid="transparent"
            />
          </View>
          <RCTScrollView
            nestedScrollEnabled={true}
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "marginBottom": 15,
                    "marginVertical": 10,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#CECECE",
                      "textAlign": "center",
                    }
                  }
                >
                  No results
                </Text>
              </View>
            </View>
          </RCTScrollView>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "height": 90,
            "justifyContent": "center",
            "marginTop": -90,
            "paddingLeft": 10,
            "width": 690,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#CECECE",
              "fontSize": 20,
            }
          }
        >
          Recipent(s)
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "#858FA1",
            "fontFamily": "Poppins-Regular",
            "fontSize": 18,
            "marginTop": 10,
            "paddingVertical": 8,
            "width": 690,
          }
        }
      >
        Subject
      </Text>
      <TextInput
        allowFontScaling={false}
        maxLength={100}
        onChangeText={[Function]}
        onSubmitEditing={[Function]}
        placeholder="Subject"
        placeholderTextColor="#CECECE"
        rejectResponderTermination={true}
        returnKeyType="next"
        style={
          Object {
            "backgroundColor": "#FFF",
            "borderColor": "#aaa",
            "borderRadius": 8,
            "borderWidth": 0.2,
            "fontFamily": "Poppins-Regular",
            "fontSize": 19,
            "height": 50,
            "paddingHorizontal": 10,
            "width": 690,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <Text
        style={
          Object {
            "color": "#858FA1",
            "fontFamily": "Poppins-Regular",
            "fontSize": 18,
            "marginTop": 10,
            "paddingVertical": 8,
            "width": 690,
          }
        }
      >
        Write Message
      </Text>
      <TextInput
        allowFontScaling={false}
        maxLength={1024}
        multiline={true}
        onChangeText={[Function]}
        placeholder="Messages"
        placeholderTextColor="#CECECE"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#FFF",
            "borderColor": "#ccc",
            "borderRadius": 8,
            "borderWidth": 0.2,
            "fontFamily": "Poppins-Regular",
            "fontSize": 19,
            "height": 155,
            "paddingHorizontal": 10,
            "textAlignVertical": "top",
            "width": 690,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <View
        style={
          Object {
            "paddingVertical": 8,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#858FA1",
              "fontFamily": "Poppins-Regular",
              "fontSize": 18,
              "marginTop": 10,
              "paddingVertical": 8,
              "width": 690,
            }
          }
        >
          What would you like to share:
        </Text>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/images/check.png",
                }
              }
              style={
                Object {
                  "height": 25,
                  "width": 25,
                }
              }
            />
            <Text
              style={
                Array [
                  Object {
                    "flex": 1,
                    "marginLeft": 10,
                  },
                  Object {
                    "color": "#858FA1",
                    "fontFamily": "Poppins-Regular",
                    "fontSize": 18,
                    "paddingVertical": 8,
                    "width": 690,
                  },
                ]
              }
            >
              Status
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/images/check.png",
                }
              }
              style={
                Object {
                  "height": 25,
                  "width": 25,
                }
              }
            />
            <Text
              style={
                Array [
                  Object {
                    "flex": 1,
                    "marginLeft": 10,
                  },
                  Object {
                    "color": "#858FA1",
                    "fontFamily": "Poppins-Regular",
                    "fontSize": 18,
                    "paddingVertical": 8,
                    "width": 690,
                  },
                ]
              }
            >
              Important Dates
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/images/uncheck.png",
                }
              }
              style={
                Object {
                  "height": 25,
                  "width": 25,
                }
              }
            />
            <Text
              style={
                Array [
                  Object {
                    "flex": 1,
                    "marginLeft": 10,
                  },
                  Object {
                    "color": "#858FA1",
                    "fontFamily": "Poppins-Regular",
                    "fontSize": 18,
                    "paddingVertical": 8,
                    "width": 690,
                  },
                ]
              }
            >
              Documents
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "paddingLeft": 60,
            }
          }
        />
      </View>
    </View>
  </RCTScrollView>
  <View
    style={
      Object {
        "backgroundColor": "#FFF",
        "height": 80,
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "#4b8eff",
          "borderRadius": 50,
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "width": "90%",
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Object {
            "color": "#FFF",
            "fontFamily": "Roboto",
            "fontSize": 20,
            "fontWeight": "bold",
            "textAlign": "center",
          }
        }
      >
        Send Email
      </Text>
    </View>
  </View>
  <Modal
    hardwareAccelerated={false}
    transparent={true}
    visible={false}
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.5)",
            "flex": 1,
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/loader1.gif",
          }
        }
        style={
          Object {
            "height": 80,
            "resizeMode": "contain",
            "width": 80,
          }
        }
      />
    </View>
  </Modal>
</View>
`;
